clang/test/3C/hash.c

```diff
@@ -9,10 +9,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-_Itype_for_any(T) void vsf_sysutil_memclr(void *p_dest
-                                          : itype(_Array_ptr<T>)
-                                                byte_count(size),
-                                            unsigned int size)
+_Itype_for_any(T) void vsf_sysutil_memclr(void *p_dest : itype(_Array_ptr<T>) byte_count(size), unsigned int size)
```

Comment:
Nit: Please change the formatting back or find some other way to keep the line length under 80 characters, as [Microsoft asked](https://github.com/microsoft/checkedc-clang/pull/993#issuecomment-791743175), even though it looks awkward.  Otherwise, I'll have to do it when I prepare the next omnibus PR to Microsoft.  I don't mind doing it, but I thought I should make you aware of the issue so maybe we can avoid changing the formatting back and forth unnecessarily.

Reply:
> As I recall, this function was originally rewritten (unnecessarily) onto one line. After the change, it is no longer rewritten and fails the unmodified check line.

Great!

> Is the solution to add multiple checks for the one function?

Yes, or more specifically, `CHECK-NEXT`.  Here's an example:
https://github.com/correctcomputation/checkedc-clang/blob/cf35b121de3211cbcac3e107edacaaf8c89e45e0/clang/test/3C/simple_locals.c#L18-L20